onchain/cairo/games/src/pixel/art_peace.cairo

```diff
@@ -453,6 +485,18 @@ pub mod ArtPeace {
         ) { // let last_shield =self.last_placed_pixel_shield.entry(pos).read();
         }
 
+        fn _has_shield(self: @ContractState, pos: u128) -> bool {
```

Comment:
We also need to call this function in the place_pixel and place_pixel_with_metadata and place_metadata too. So if no shield active user can place a pixel, otherwise no